examples/prefix/prefix.go

```diff
@@ -0,0 +1,22 @@
+package prefix
+
+import (
+	"github.com/11wizards/go-to-dart/examples/prefix/shared"
+)
+
+type KeyValuePair[TKey, TValue any] struct {
+	Key   TKey   `json:"key"`
+	Value TValue `json:"value"`
+}
+
+type Map[TKey, TValue any] struct {
+	Items []KeyValuePair[TKey, TValue] `json:"items"`
+}
+
+type Instance struct {
+	M Map[string, int] `json:"m"`
+}
+
+type UserRepository struct {
+	Users []shared.User `json:"users"`
```

Comment:
shared.User comes from a different package and should NOT be prefixed in the generated code.